src/Onboarding/EulaModal.tsx

```diff
@@ -225,6 +219,13 @@ const styles = StyleSheet.create({
   checkboxText: {
     ...Forms.checkboxText,
   },
+  activityIndicator: {
```

Comment:
I believe there's a PR #31 that works on this it might be worth merging that design with this?